test/e2e/apimachinery/webhook.go

```diff
@@ -709,6 +717,100 @@ func testFailClosedWebhook(f *framework.Framework) {
 	}
 }
 
+func registerWebhookForWebhookConfigurations(f *framework.Framework, context *certContext) func() {
+	client := f.ClientSet
+	By("Registering a webhook on ValidatingWebhookConfiguration objects, via the AdmissionRegistration API")
+
+	namespace := f.Namespace.Name
+	configName := webhookForWebhooksConfigName
+	failurePolicy := v1beta1.Fail
+
+	// This broken webhook will deny all requests to Delete ValidatingWebhookConfiguration objects
+	_, err := client.AdmissionregistrationV1beta1().ValidatingWebhookConfigurations().Create(&v1beta1.ValidatingWebhookConfiguration{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: configName,
+		},
+		Webhooks: []v1beta1.Webhook{
+			{
+				Name: "deny-validating-webhook-configuration-deletions.k8s.io",
+				Rules: []v1beta1.RuleWithOperations{{
+					Operations: []v1beta1.OperationType{v1beta1.Delete},
+					Rule: v1beta1.Rule{
+						APIGroups:   []string{"admissionregistration.k8s.io"},
+						APIVersions: []string{"*"},
+						Resources:   []string{"validatingWebhookConfiguration"},
+					},
+				}},
+				ClientConfig: v1beta1.WebhookClientConfig{
+					Service: &v1beta1.ServiceReference{
+						Namespace: namespace,
+						Name:      serviceName,
+						// NOTE: This path isn't something the service knows how to handle.
+						Path: strPtr("/validating-webhook-configurations"),
+					},
+					// Without CA bundle, the call to webhook always fails
```

Comment:
Could you add to the comment that the failure policy is "Fail"? Otherwise this test would have given us false negative.

Reply:
I am working on changing this to use the correct CA and updating the webhook test image to handle a function that always denies requests so this test doesn't have to rely on a broken CA and failing closed to deny requests.
I think that will be a better test.